tests/console/rails.pm

```diff
@@ -14,21 +14,25 @@ use strict;
 use warnings;
 use base "consoletest";
 use testapi;
+use version_utils "is_jeos";
+use utils;
 
 sub run {
     select_console 'root-console';
+    zypper_call("in gcc make git zlib-devel ruby-devel sqlite3-devel nodejs") if is_jeos;
```

Comment:
And please drop all these as well - they are not supposed to be required for rails stack.

Reply:
These are packages it complained about when it didn't have a header or a command was missing. If I take them out there is no point to the PR so I will just close it.